tests/benchmarks/model_benchmarks/test_pytorch_base.py

```diff
@@ -174,62 +174,58 @@ def test_pytorch_base():
     # Register BERT Base benchmark.
     BenchmarkRegistry.register_benchmark('pytorch-mnist', PytorchMNIST)
 
-    # Launch benchmark for testing.
-    context = BenchmarkContext(
+    # Launch benchmark with --no_gpu for testing.
+    context = BenchmarkRegistry.create_benchmark_context(
         'pytorch-mnist',
-        Platform.CPU,
```

Comment:
Do we need to add `platform=Platform.CPU,`

Reply:
Done.